public/es6.html

```diff
@@ -112,9 +114,6 @@ <h2 class="faq-title">
       <div class="faq-body">
         <p>io.js provides a simple way to list all dependencies and respective versions that ship with a specific binary through the <code>process</code> global object. In case of the V8 engine, type the following in your terminal to retrieve its version:</p>
         <div class="highlight highlight-sh"><pre>iojs -p process.versions.v8</span></pre></div>
-
-        <p>E.g. output: <code>3.31.74.1</code>.</p>
```

Comment:
I wonder if we should leave the example output, but just put in pseudo-numbers so people don't think that's the current version. To me an example !== what I am going to see (in my local version) anyway.